pkg/route/ingress/ingress.go

```diff
@@ -467,6 +492,13 @@ func (c *Controller) sync(key queueKey) error {
 		}
 	}
 
+	defer func() {
+		if len(errs) == 0 && len(IngressToRouteRule) > 0 {
+			message := fmt.Sprintf("Failed ingress to route rules detected in ingress %s/%s: %s", ingress.Namespace, ingress.Name, strings.Join(IngressToRouteRule, "; "))
+			c.eventRecorder.Eventf(ingress, corev1.EventTypeNormal, "IngressToRouteRule", message)
```

Comment:
```suggestion
			c.eventRecorder.Eventf(ingress, corev1.EventTypeNormal, "IngressToRouteRule", "Incomplete ingress to route rules detected: %s", strings.Join(IngressToRouteRule, "; "))
```

1. The `ingress` object should be enough to show which ingress had the rules, right? We do this way for the warning event.
2. I think the "incomplete" wording which you proposed would be slightly more relaxed for the user. Sorry about back and forth, this one is hard to name perfectly.